ando/AnDOChecker.py

```diff
@@ -7,24 +7,28 @@
 
 
 def is_valid(input_directory):
-    """Checks the validity of a data set with respect to the BIDS-animal-ephys specifications.
-
+    """
+    Checks the validity of a data set with respect to the BIDS-animal-ephys specifications.
     The specifications that define what is checked by this function is available in the following document:
-    https://docs.google.com/document/d/1oG-C8T-dWPqfVzL2W8HO3elWK8NIh2cOCPssRGv23n0
+    https://bids.neuroimaging.io/bep032
 
 
     Parameters
     ----------
-        input_directory : string
+    input_directory : string
             Name of the root directory containing the data set to be checked
 
     Returns
     -------
-        valid : boolean
-            True if the data set follows the ephys-BIDS specification; False if not
+    A Tuple of size 2 with the first a
+
+    boolean
+        True if the data set follows the ephys-BIDS specification; False if not
```

Comment:
```suggestion
    tuple
        a tuple of size 2 containing
        boolean
            True if the data set follows the ephys-BIDS specification; False if not
        list
            List of errors (empty if the data set is valid)
```

Reply:
yes this one is tricky because for some reason Sphinx put a bullet point at the beginning of each line like 
![image](https://user-images.githubusercontent.com/17256769/114679614-9b8fe800-9d0c-11eb-92d4-ea7baa17db09.png)